src/lib/components/workers-table.svelte

```diff
@@ -0,0 +1,17 @@
+<section
```

Comment:
Do you think we benefit from breaking this out to its own component instead of just combining this and `workers-row.svelte` into one component?

I guess my other question is should these be in the `$lib/components` considering they're dedicated to one particularly page—like the workflows summary table.